deps/fdasrsf/dp_grid.c

```diff
@@ -0,0 +1,284 @@
+#include <stdio.h>
```

Comment:
I was thinking if we could standardize a C format style and apply it before the first commit. In particular, I would like 4 spaces (or a tab) per indentation level, spaces around operators, no spaces around parentheses...

Reply:
Done in 396600a.